src/pages/guides/getting_started/code_playground.md

```diff
@@ -12,54 +14,223 @@ keywords:
-</div><br/>
+</div><br/> -->
 
-## Features
+## Development Workflow Use Cases
+
+There are numerous opportunities for using the Code Playground to help you throughout the development process, including:
 
-- **Real-Time Preview:** See your changes instantly as you code, allowing for immediate feedback and faster adjustments.
-- **Effortless Prototyping:** Quickly turn ideas into add-ons with minimal setup.
-- **Rapid Implementation:** Fast-track your prototype to a product by directly pasting your code into an add-on template.
+- **Experiment First**: Test your ideas and API interactions before committing to full add-on development.
+- **Learn as You Go**: Master the basics of Document APIs and Add-on SDK without complex setup requirements.
+- **Prototype Quickly**: Build and test features in minutes instead of hours with instant feedback.
+- **Bridge to Production**: Develop core functionality in Playground before moving to a complete project environment.
+- **Debug with Ease**: Isolate and fix specific issues by testing API calls outside your production code.
 
-## Getting Started
+## How to Access Code Playground
 
-### Step 1: Enable Add-On Development Mode
+### Step 1: Enable Add-on Development Mode
 
-Go to your settings and enable add-on development mode (if it isn't already enabled).
+- Open Adobe Express settings by clicking the gear icon
+- Enable "Add-on Development Mode" if not already enabled
 
 ![Adobe Express Settings](./img/settings_alt.png)
 
-### Step 2: Locate the 'Code-playground' option
+### Step 2: Open Code Playground
+
+- With any document open, click the "Add-ons" button in the left sidebar
+- Select the "Your add-ons" tab
+- Toggle on "Code Playground" at the bottom of the panel
+
+  ![Adobe Express Code Playground Toggle](./img/toggle-playground.png)
+
+- Once enabled, the playground window will open, allowing you to begin coding immediately!
+
+  ![Adobe Express Code Playground](./img/script-mode.png)
+  
+## Choose Your Development Mode
+
+The playground offers two distinct development modes, each designed for different development needs:
+
+### Comparison of Modes
+
+The playground offers two distinct development modes, each designed for different development needs:
+
+| Feature | Script Mode | Add-on Mode |
+|---------|-------------|-------------|
+| **Purpose** | Quick document manipulation tests | Complete add-on UI and functionality |
+| **Environment** | Document Sandbox only | Both iframe and Document Sandbox |
+| **API Access** | Document APIs | Document APIs + Add-on UI SDK |
+| **UI Components** | No UI building | Full HTML/CSS/JS interface creation |
+| **Best For** | Testing document operations | Building complete add-ons |
+
+### When to Use Script Mode
+
+- When focusing on document manipulation operations
```

Comment:
I also feel that while we are calling out that Script mode is for Document APIs, we should call out that its equivalent to `sandbox/code.js`. I say so because I observe that we mention `sandbox/code.js` in a lot of code snippets. For example here: https://developer.adobe.com/express/add-ons/docs/guides/develop/how_to/use_color/#create-colors